source/integrate_with_govuk_pay/index.html.md.erb

```diff
@@ -43,8 +43,7 @@ You will likely need some kind of server-side datastore to record payment inform
 
 ## The GOV.UK Pay API
 
-The GOV.UK Pay API offers a set of operations to conduct and report on
-payments. For more information:
+The GOV.UK Pay API offers a set of endpoints to help your organisation reconcile and report on payments. For more information:
```

Comment:
would integrate work instead of reconcile? Again feel free to ignore me just noodling words around in my head to think how they would sound!

Reply:
So reconciliation is a payment-specific thing. It's the process of matching the money going into your bank account to the payment entries in your database or whatever. It's not the clearest language though, completely agree. Don't think it can go anywhere here, though :'(

Reply:
I thought this would be the case with both things but thought it worth raising! The user will understand and that is the main thing isn't it!